src/core/Menu/MenuItem.tsx

```diff
@@ -103,7 +114,10 @@ export const MenuItem = React.forwardRef<HTMLLIElement, MenuItemProps>(
           React.cloneElement(icon, {
             className: cx(icon.props.className, 'iui-icon'),
           })}
-        <span className='iui-content'>{children}</span>
+        <span className='iui-content'>
+          <div className='iui-menu-label'>{children}</div>
+          {sublabel && <div className='iui-menu-description'>{sublabel}</div>}
```

Comment:
There is a limitation from css, that if it is not large, sublabel wont shown? Cause you do not check it in code here

Reply:
Yeah, it's set in css